packages/api/src/pins.js

```diff
@@ -84,24 +143,43 @@ export async function pinPost (request, env, ctx) {
  * @param {import('./index').Ctx} ctx
  */
 export async function pinGet (request, env, ctx) {
-  const requestId = request.params.requestId
-
-  if (!requestId) {
+  let requestId
+  try {
+    requestId = parseInt(request.params.requestId, 10)
```

Comment:
From the spec it looks like this can be a string (i.e. "requestid": "UniqueIdOfPinRequest"), are we sure we can always coerce this into an int?

Reply:
Good catch 👀 , however we're using the PinRequest primary key as a unique id which, and that is set to be a `bigserial`. So in our system, as it stands right now, the id has to be a coercible int.
Am I missing something?

Reply:
100% agreed on the db side, but if you parseInt a string, it'll return `NaN` and so I guess it'll just fail in the next try/catch? Is that what we want?

Reply:
I was indeed missing something! It'd be better to have it just fail here.
Let me update this

Reply:
done ✅